app/services/save_practice_service.rb

```diff
@@ -145,6 +145,55 @@ def update_category_practices
     category_params = @practice_params[:category]
     practice_category_practices = @practice.category_practices
     practice_categories = @practice.categories
+    categories_selected = Array.new
+    categories_unselected = Array.new
+    dm_notification_categories = ["COVID", "TELEHEALTH", "CURBSIDE CARE", "ENVIRONMENTAL SERVICES", "PULMONARY CARE", "HEALTHCARE ADMINISTRATION"]
+
+
+    if category_params.blank? && practice_categories.present?
+      practice_categories.each do |pc|
+        categories_unselected.push(pc.id)
```

Comment:
Could we consolidate some of the logic here so instead of pushing it to one array and then iterating over that array again?
- We could get an array of the `dm_notification_categories` by finding `Category` with that `name` then pushing that `Category` `id` into that array.
- Then we can check if the id is a covid category before pushing it in.
Let me know  what you think, Brad!

Reply:
I see what you mean Elisa... use the category IDs instead of name but I'm doing a contains on "COVID" to catch any future covid related categories...  plus there are only 6 covid related categories so it's not a performance issue.. I agree if we were comparing 1000s of strings in a list.. then using array of ids would be much faster but you'd still have the upfront work of creating the array of category IDs..